### PR TITLE
refactor code as per pep8 using flake8

### PR DESCRIPTION
**Pull request by vaibhavmule**
used `flake8 masonite/ --ignore=E501,F401,E128,E402,E731,F821,E712 `

### CODE REVIEW

File: masonite/providers/AppProvider.py [hunk 1]
```diff
@@ -73,9 +73,9 @@ def boot(self, request: Request, route: Route):
...
         Autoload(self.app).load(directories)
-    
+
     def _set_application_debug_level(self):
-        if self.app.make('Application').DEBUG == 'True':
-            self.app.make('Application').DEBUG == True
-        elif self.app.make('Application').DEBUG == 'False':
-            self.app.make('Application').DEBUG == False
+        if self.app.make('Application').DEBUG is 'True':
+            self.app.make('Application').DEBUG is True
```
Comment (vaibhavmule) [inline masonite/providers/AppProvider.py]: @josephmancuso Your opinion?

Comment (josephmancuso) [reply masonite/providers/AppProvider.py]: or is that not true?

Comment (josephmancuso) [reply masonite/providers/AppProvider.py]: nvm. That's not true. yeah i'm a fan of doing it like that

Comment (vaibhavmule) [reply masonite/providers/AppProvider.py]: This is what best explanation I can find. 
`is` checks that 2 arguments refer to the same object, `==` checks that 2 arguments have the same value.

So, I need to change that back to `==` for this line `if self.app.make('Application').DEBUG is 'True':`

Comment (josephmancuso) [reply masonite/providers/AppProvider.py]: ok that's what I thought lol

**Pull request merged**